tests/testing.md

```diff
@@ -0,0 +1,66 @@
+@page testing Testing Guide
+
+[TOC]
+
+[Integration testing](https://en.wikipedia.org/wiki/Integration_testing) is an essential part of software development. Essentially, assume that every untested line in `svFSIplus` is not working correctly. Below is a quick guide on how to run and add integration tests for `svFSI`.
```

Comment:
I suggest replacing
```
Essentially, assume that every untested line in `svFSIplus` is not working correctly.`
```

with something like
```
Integration testing is performed when integrating code changes into the main development branch to verify that the code works as expected.

Reply:
Haha, I'll change it! My other idea was "always be vigilant"